src/opentimelineview/console.py

```diff
@@ -34,6 +34,7 @@
 import opentimelineio.console as otio_console
 import opentimelineview as otioViewWidget
 from opentimelineview import settings
+from ges_renderer import Renderer
```

Comment:
I'd prefer not to do a `from` import for `Renderer`, I think it would read more clearly below if the branch read:

`if ges_renderer.Renderer is not None`

Reply:
Done.